x-pack/plugins/observability_solution/synthetics/e2e/synthetics/journeys/alert_rules/custom_status_alert.journey.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { journey, step, before, after, expect } from '@elastic/synthetics';
+import { RetryService } from '@kbn/ftr-common-functional-services';
+import { syntheticsAppPageProvider } from '../../page_objects/synthetics_app';
+import { SyntheticsServices } from '../services/synthetics_services';
+
+journey(`CustomStatusAlert`, async ({ page, params }) => {
```

Comment:
I'm gonna be honest. I no longer really love the idea of continuing to write journeys as a replacement for functional tests. Here's why: 

Custom journey tests do not benefit from the added value that the app ex qa team adds to traditional functional and  api integration tests, including running the tests in cloud against ESS and MKI instances.

Previously, while the Synthetics product was less mature, we benefited from dogfooding synthetics agent. Now that we have a mature product, I feel like we benefit less from using the Synthetics agent in a way it's not really intended, for testing rather than monitoring.

Reply:
i think that's a topic that we can discuss, but we need a plan to move these into functional test runner.